concepts/floating-point-numbers/about.md

```diff
@@ -0,0 +1,25 @@
+# About
```

Comment:
I am making a number of style comments in this about.md file that apply to the introduction files as well.